app/src/main/res/layout-land/ongoing_story_card.xml

```diff
@@ -12,12 +12,9 @@
   </data>
```

Comment:
In this the opacity of `topic_name_text_view ` is `80%` so the alpha should be `0.8` or the color should be changed accordingly.

Reply:
`story_name_text_view ` should have max & min Lines as `2`.

Reply:
Done.